javascript/javascript2/week3/game/game.js

```diff
@@ -0,0 +1,79 @@
+const btnTag = document.getElementById("start-btn");
+const restartBtnTag = document.getElementById("restart-btn");
+const durationInput = document.getElementById("set-time");
+const sCountInput = document.getElementById("s-count");
+const lCountInput = document.getElementById("l-count");
+const result = document.getElementById("result");
+const confettiElements = document.getElementsByClassName("confetti-canvas");
+let gameRunning = false;
+
+function resetCount() {
+  sCount = 0;
```

Comment:
do you need let, var, or const here. read about difference b/w defining value witout let, var etc